layouts/partials/footer.html

```diff
@@ -26,11 +26,22 @@ <h5>Resources</h5>
                 </ul>
             </div>
             <div class="col-6 col-md">
-                <h5>About</h5> Our code is free and 100% MIT-licensed; even this website!
+				<h5>dApp  Competitions</h5>
+                <ul class="list-unstyled text-small">
+                    <li><a class="text-muted" href="/dapps/1_launch/">Competition #1</a></li>
+                    <li><a class="text-muted" href="/dapps/1/">Winners Competition #1</a></li>
+                    <li><a class="text-muted" href="/dapps/2_launch/">Competition #2</a></li>
+                </ul>	            
             </div>
             <div class="col-6 col-md">
                 <h5>Help</h5> If you feel you can improve on anything here, just send us a <a href="https://github.com/CityOfZion/cityofzion-website">pull request</a> ❤️
             </div>
+            <div class="col-xl-12">
+				<center>
```

Comment:
indentation

Reply:
It's weird, in the code editor is properly indented. I updated the code now, i looks like github is showing too much tabs.